### PR TITLE
Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-generic.git`](https://github.com/cisagov/skeleton-generic.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

## ✅ Pre-approval checklist ##

Remove any of the following that do not apply. If you're unsure about
any of these, don't hesitate to ask. We're here to help!

- [x] All relevant type-of-change labels have been added.
- [x] All new and existing tests pass.

---

> **Note**
> You are seeing this because one of this repository's maintainers has
> configured [Lineage] to open pull requests.

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -47,13 +47,16 @@ jobs:
       - id: setup-python
         uses: actions/setup-python@v4
         with:
-          python-version: "3.10"
+          python-version: "3.11"
```
Comment (mcdonnnj) [inline .github/workflows/build.yml:50]: I believe this should be done to the `test` job as well:

https://github.com/cisagov/skeleton-docker/blob/201692f8e35feff3badc715fc74d7bef1e3ba308/.github/workflows/build.yml#L306-L309

**Review by dav3r — APPROVED** [review]
👍

**Review by jsf9k — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (mcdonnnj) [reply .github/workflows/build.yml:50]: _I'll do it myself_ [18c3bc4](https://github.com/cisagov/skeleton-docker/pull/164/commits/18c3bc4cd9251631448e089bd7baa3bccc6694dc).

**Review by mcdonnnj — APPROVED** [review]
Ok _this time_ I think we're good.

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
☸️

**Pull request merged**